hummedia/api.py

```diff
@@ -35,6 +35,17 @@ def Collection(collection,id=None):
 	else:
 		return endpoint_404()
 
+@app.route('/video/stale', methods=['GET', 'OPTIONS'])
+@app.route('/video/stale/<date>', methods=['GET', 'OPTIONS'])
+@crossdomain(origin=CROSS_DOMAIN_HOSTS,headers=['Origin','x-requested-with','accept','Content-Type'],credentials=True)
```

Comment:
I don't think a GET request with Angular's $http resource will send a 'Content-Type' header, so you can probably get rid of its declaration here.